tests/test_examples.py

```diff
@@ -692,7 +691,7 @@ def test_NAE_QIC_solve():
     lam_nae = np.squeeze(lam_nae[:, 0, :])
 
     lam_av_nae = np.mean(lam_nae, axis=0)
-    np.testing.assert_allclose(lam_av_nae, -qsc.iota * qsc.nu_spline(phi), atol=1e-4)
+    np.testing.assert_allclose(lam_av_nae, -qsc.iota * qsc.nu_spline(phi), atol=5e-4)
```

Comment:
I think the tolerances were too tight. This regression test would pass on GitHub CI, but failed locally.
[QIC test failure log.txt](https://github.com/PlasmaControl/DESC/files/12139032/QIC.test.failure.log.txt)